scripts/tests/chiptest/__init__.py

```diff
@@ -160,6 +160,59 @@ def _GetInDevelopmentTests() -> Set[str]:
     }
 
 
+def _GetDarwinFrameworkToolUnsupportedTests() -> Set[str]:
+    """Tests that fail in darwin-framework-tool for some reason"""
+    return {
+        "DL_LockUnlock",  # Events error.
+        "Test_AddNewFabricFromExistingFabric",  # darwin-framework-tool does not support the GetCommissionerRootCertificate command.
+        "TestAttributesById",
+        "TestBasicInformation",  # Test Failure : The test expects the "UNSUPPORTED_WRITE" error but the "FAILURE" error occured
+        "TestClusterComplexTypes",
+        "TestClusterMultiFabric",
+        "TestCommandsById",  # Test Failure : The test expects the "UNSUPPORTED_ENDPOINT" error but the "FAILURE" error occured.
+        "TestCommissioningWindow",
+        "TestDiagnosticLogs",  # darwin-framework-tool does not implement a BDXTransferServerDelegate
+        "TestDiscovery",  # darwin-framework-tool does not support dns-sd commands.
+        "TestEvents",
+        "TestEventsById",
+        "TestGroupMessaging",  # darwin-framework-tool does not support group commands.
+        "TestIcdManagementCluster",  # darwin-framework-tool does not support ICD registration
+        "TestUnitTestingClusterMei",
+        "TestReadNoneSubscribeNone",  # darwin-framework-tool does not supports those commands.
+
+        "Test_TC_ACE_1_6",  # darwin-framework-tool does not support group commands.
+        "Test_TC_ACL_2_5",  # Events error.
+        "Test_TC_ACL_2_6",  # Events error.
+        "Test_TC_ACL_2_7",  # Events error.
+        "Test_TC_ACL_2_8",  # Events error.
+        "Test_TC_ACL_2_9",  # Events error.
+        "Test_TC_ACL_2_10",  # Events error.
+        "Test_TC_BINFO_2_1",  # Test Failure : The test expects the "UNSUPPORTED_WRITE" error but the "FAILURE" error occured
+        "Test_TC_BINFO_2_2",  # Events error.
+        "Test_TC_BRBINFO_2_1",  # Test Failure : The test expects the "UNSUPPORTED_ATTRIBUTE" error but the "FAILURE" error occured.
+        "Test_TC_DGEN_2_1",  # minValue: 1 <-- The response value (0) should be greater or equal to the constraint but 0 < 1
```

Comment:
So how does this pass anywhere else?

Reply:
The test `Test_TC_DGEN_2_1` does not seems to exist anymore... I am removing the line